app/networkapi/people/models.py

```diff
@@ -92,6 +106,20 @@ class Person(models.Model):
         help_text='Do you want to feature this person?',
         default=False,
     )
+    publish_after = models.DateTimeField(
+        help_text='Make this person\'s profile visible only after '
+                  'this date and time (UTC)',
+        default=timezone.now,
```

Comment:
no because it's a static field in the model, there is a param you can pass `auto_now_add` which is basically specifying the default, but it makes the form field itself non-editable, which sucks.
I opened up a pr to fix this (the trick is to do it in the form instead)